observability/otel/otel-updating.adoc

```diff
@@ -1,6 +1,6 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="dist-tracing-otel-updating"]
-= Updating the {OTELShortName}
+= Upgrading
```

Comment:
Global comment, but jw why did you remove the other part of this? In the mod docs guide, I read that title should be a gerund phrase: https://redhat-documentation.github.io/modular-docs/#assembly-guidelines

Reply:
@skrthomas, I previously received peer reviewer feedback suggesting that I shorten the headings for a number of reasons: (1) make headings uniform between the page and the nav entry, (2) reduce fluff (these product name attributes expand into long multiple-word product names), and 3) because there is already context and inline mentions that identify the product/component that the page is about, so it's not necessary to add the product/component name to every heading.

Reply:
Thanks for clarifying this. Apologies for providing contradictory peer review. I noticed the context..since you already altered the titles due to a peer review suggestion, you can disregard. This is maybe something peer reviewers should sync on.